since r59150 wxXmlResource::LoadFile() uses wxFileSystem internally,
so enclose declaration and implementation within #if wxUSE_FILESYSTEM

Signed-off-by: Anders Larsen <al@alarsen.net>
---
 include/wx/xrc/xmlres.h |    2 ++
 src/xrc/xmlres.cpp      |    4 ++--
 2 files changed, 4 insertions(+), 2 deletions(-)

Index: b/include/wx/xrc/xmlres.h
===================================================================
--- a/include/wx/xrc/xmlres.h
+++ b/include/wx/xrc/xmlres.h
@@ -112,11 +112,13 @@ public:
     // This method understands wxFileSystem URLs if wxUSE_FILESYS.
     bool Load(const wxString& filemask);
 
+#if wxUSE_FILESYSTEM
     // Loads resources from single XRC file.
     bool LoadFile(const wxFileName& file);
 
     // Loads all XRC files from a directory.
     bool LoadAllFiles(const wxString& dirname);
+#endif // wxUSE_FILESYSTEM
 
     // Unload resource from the given XML file (wildcards not allowed)
     bool Unload(const wxString& filename);
Index: b/src/xrc/xmlres.cpp
===================================================================
--- a/src/xrc/xmlres.cpp
+++ b/src/xrc/xmlres.cpp
@@ -176,8 +176,6 @@ bool wxXmlResource::IsArchive(const wxSt
     return fnd.Matches(wxT("*.zip")) || fnd.Matches(wxT("*.xrs"));
 }
 
-#endif // wxUSE_FILESYSTEM
-
 bool wxXmlResource::LoadFile(const wxFileName& file)
 {
     return Load(wxFileSystem::FileNameToURL(file));
@@ -199,6 +197,8 @@ bool wxXmlResource::LoadAllFiles(const w
     return ok;
 }
 
+#endif // wxUSE_FILESYSTEM
+
 bool wxXmlResource::Load(const wxString& filemask_)
 {
     wxString filemask = ConvertFileNameToURL(filemask_);
